example-apps/session-auth/index.js

```diff
@@ -17,6 +17,8 @@ const sessionRefreshIf401 = (response, z, bundle) => {
       throw new z.errors.RefreshAuthError('Session key needs refreshing.');
     }
   }
+  // When you define an `afterResponse`, make sure one of them calls `throwForStatus`
+  response.throwForStatus();
```

Comment:
Updated examples that use `afterResponse` to show that they are now responsible for calling `throwForStatus`. Of course they could also do this in their perform-like functions but this is the preferred way.